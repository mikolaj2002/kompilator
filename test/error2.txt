# Error2. Use of uninitialized variable d in line 7

PROCEDURE pa(T a, b) IS
 c[5],d
IN
  c[b]:=b;
  a[d]:=c[d];
END

PROGRAM IS
 a[10],b
IN
  READ b;
  pa(a,b);
  WRITE a[b];
END
