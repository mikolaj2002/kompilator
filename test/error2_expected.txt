[COMPILER] Warning: Variable d might not be initialized, used in line 7
[COMPILER] Warning: Variable d might not be initialized, used in line 7
